vNext/samples/ODataSample.Web/Controllers/CustomersController.cs

```diff
@@ -26,9 +26,22 @@ public IEnumerable<Customer> Get()
 
         // GET api/Customers/5
         [HttpGet("{id}")]
-        public IActionResult Get(int id)
+        public IActionResult Get(int customerId)
```

Comment:
Can request route to this method?  Do you also need to update the route template to '[HttpGet("{customerId}")]`?